qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -259,12 +259,14 @@ def aqt_compile(
     def ibmq_compile(
         self,
         circuits: Union[qiskit.QuantumCircuit, Sequence[qiskit.QuantumCircuit]],
+        dynamical_decoupling: bool = False,
         **kwargs: Any,
     ) -> qss.compiler_output.CompilerOutput:
         """Compiles and optimizes the given circuit(s) for IBMQ devices.
 
         Args:
             circuits: The qiskit QuantumCircuit(s) to compile.
+            dynamical_decoupling: Applies dynamical decoupling optimization to circuit.
```

Comment:
minor nit if dd is applied to all circuits when a list of circuits is passed into `ibmq_compile`:
```suggestion
            dynamical_decoupling: Applies dynamical decoupling optimization to the circuit(s).
```

Reply:
Great point! Will make the change!